package.json

```diff
@@ -650,6 +650,16 @@
                 "command": "explorer.newFolder",
                 "when": "filesExplorerFocus && !inputFocus"
             },
+            {
+                "key": "c",
+                "command": "filesExplorer.copy",
+                "when": "explorerViewletVisible && filesExplorerFocus && !explorerResourceIsRoot && !inputFocus"
+            },
+            {
+                "key": "shift+w",
+                "command": "workbench.files.action.collapseExplorerFolders",
+                "when": "filesExplorerFocus"
```

Comment:
`zM` would be more appropriate here. Or `zc` if this is cursor-relative.

Reply:
I wasn't aware of Neovim's fold mechanism. I agree about `zM` after reading it's documentation. For some reason `nvim-tree` uses `W`. Interested in supporting both maps? I could add `zM` to my PR.

Reply:
`W` seems random. Bad defaults chosen by plugins shouldn't leak into upstream projects.